pkg/controller/volume/persistentvolume/config/v1alpha1/defaults.go

```diff
@@ -39,6 +39,11 @@ func RecommendedDefaultPersistentVolumeBinderControllerConfiguration(obj *kubect
 		obj.PVClaimBinderSyncPeriod = metav1.Duration{Duration: 15 * time.Second}
 	}
 
+	if obj.VolumeHostAllowLocalLoopback == nil {
+		trueValue := true
+		obj.VolumeHostAllowLocalLoopback = &trueValue
```

Comment:
is the intent to default this to true to match current behavior, pick that change back to 1.17/1.18, then flip the default to false for 1.20+?

Reply:
Yes, I think so.

jsafrane above mentioned default-false would break some of his test setups, no-one was sure about real users. Out of an abundance of caution then we decided to keep it opt-in until we get something more definite.

I'm happy to hear another opinion, especially if it's more decisive!